### DISCUSSION TITLE
SherwoodBank

### DISCUSSION THREAD

**Discussion by passandscore** [passandscore/solidity-challenges#17] [Exploits]
# SherwoodBank

SherwoodBank, a widely trusted contract, has recently come under scrutiny due to a potential security flaw. Your mission: identify the vulnerability and drain all the funds from the contract before anyone else does.

Think you have what it takes? Prove it.

**Author**: [IzuMan](https://github.com/IzuMan0x)

**Hints**:
Run a static analyzer on the contract like slither or aderyn which will point out the contract's flaw.

**Comment by IzuMan0x** [comment]
A big hint is to look at how `msg.value` is used in a loop.🤫
